src/main/java/mate/academy/rickandmorty/controller/CharacterController.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.CharacterDto;
+import mate.academy.rickandmorty.service.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/characters")
+public class CharacterController {
+
+    private final CharacterService characterService;
+
+    @Tag(name = "Get")
+    @GetMapping()
+    @Operation(summary = "Get random character", description = "Get random wiki about one "
+            + "character in the universe the animated series Rick & Morty")
+    public CharacterDto get() {
```

Comment:
Rename it to reflect behavior.
```suggestion
    public CharacterDto random() {
```